rust/lance-index/src/scalar/inverted/index.rs

```diff
@@ -329,11 +329,10 @@ impl ScalarIndex for InvertedIndex {
         query: &dyn AnyQuery,
         _metrics: &dyn MetricsCollector,
     ) -> Result<SearchResult> {
-        let query = query.as_any().downcast_ref::<SargableQuery>().unwrap();
-        return Err(Error::invalid_input(
-            format!("unsupported query {:?} for inverted index", query),
-            location!(),
-        ));
+        // TODO: use fts for search
+        let _query = query.as_any().downcast_ref::<TextQuery>().unwrap();
+        let values: Vec<u64> = vec![];
+        Ok(SearchResult::AtLeast(RowIdTreeMap::from_iter(values)))
```

Comment:
This is maybe jumping ahead of ourselves.  Let's include this change in the future PR when we can actually test it?